vllm/lora/models.py

```diff
@@ -637,18 +605,16 @@ def create_lora_manager(
         max_num_batched_tokens: int,
         vocab_size: int,
         lora_config: LoRAConfig,
-        target_modules: Union[str, List[str]] = TARGET_MODULES_QKV,
         lora_manager_cls: Type[LoRAModelManager] = LoRAModelManager,
         **kwargs) -> LoRAModelManager:
     """Create a LoRA adapter for a given model."""
-    if not getattr(model, "supports_lora", False):
+    if not getattr(model, "supported_lora_modules", False):
```

Comment:
Nit: This is slightly odd, `if not hasattr(model, "supported_lora_modules")` would is more natural here, since `supported_lora_modules` is not of type bool :)